modules/tutorials/examples/logging-aggregator/main.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+echo "Installing ZooKeeper Operator"
+# tag::zk-op[]
+#stackablectl release install -i secret -i commons -i zookeeper 23.7
+stackablectl op in secret commons zookeeper
+# end::zk-op[]
+
+# tag::vector-agg[]
+helm install --wait -f vector-aggregator-values.yaml vector-aggregator vector/vector
```

Comment:
On large screens, only 75 characters are shown per line. To avoid that the user has to scroll horizontally in a code block, this limit should not be exceeded.

Perhaps it makes sense to indent the lines only by two spaces to save even more space (especially on mobile devices).

I also replaced `-f` with `--values`. It takes more space but long names are often more readable.

```suggestion
helm install \
  --wait \
  --values vector-aggregator-values.yaml \
  vector-aggregator vector/vector
```